install.sh

```diff
@@ -0,0 +1,7 @@
+mkdir tmp
```

Comment:
filename should indicate this is just for use on travis

also it needs some explanatory comment inside the file